src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataLatestIT.java

```diff
@@ -712,4 +712,30 @@ public void testHandlingSpecialChars() throws SQLException {
     assertTrue(resultSet.next());
     assertEquals("COLA", resultSet.getString("COLUMN_NAME"));
   }
+
+  /**
+   * This tests that the driver doesn't add quotes to the database name if the application has
+   * already added them. This fixes a bug where the show schemas command was returning empty results
+   * since the database name was being double-quoted.
+   *
+   * @throws SQLException
+   */
+  @Test
+  public void testGetSchemasWithDoubleQuotedDatabase() throws SQLException {
```

Comment:
This test should be moved to DatabaseMetaDataInternalLatestIT (has internal in the name), because it looks like this test is failing due to lack of permissions needed to create a database. It should work inside the internal file, however.